frameworks/CSharp/FastHttpApi/PlatformBenchmarks/HttpParse.cs

```diff
@@ -0,0 +1,102 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace PlatformBenchmarks
+{
+    public class HttpParse
+    {
+        static HttpParse()
+        {
+            LineBytes = Encoding.UTF8.GetBytes("\r\n");
+
+            NotFoundBytes = Encoding.UTF8.GetBytes("HTTP/1.1 404 not found\r\n\r\n");
+
+            StringBuilder sb = new StringBuilder();
+            sb.AppendLine("HTTP/1.1 200 OK");
+            sb.AppendLine("Content-Type: text/plain");
+            sb.AppendLine("Content-Length: 14");
+            sb.AppendLine("Date: " + DateTime.Now.ToUniversalTime().ToString("r"));
+            sb.AppendLine("");
+            sb.Append("BeetleX Server");
+            BeetleXServerBytes = Encoding.UTF8.GetBytes(sb.ToString());
+            sb.Clear();
+
+            //PlaintextBytes
+            string helloword = "Hello, World!";
+            sb.AppendLine("HTTP/1.1 200 OK");
+            sb.AppendLine("Content-Type: text/plain");
+            sb.AppendLine("Content-Length: " + helloword.Length);
+            sb.AppendLine("Server: BeetleX");
+            sb.AppendLine("Date: " + DateTime.Now.ToUniversalTime().ToString("r"));
+            sb.AppendLine("");
+            sb.Append(helloword);
+            PlaintextBytes = Encoding.UTF8.GetBytes(sb.ToString());
```

Comment:
I don’t think this is legal. You need to create the response every single request, you can’t cache it.

Reply:
@IKende please look at the test requirements. For the json test, you must instantiate and serialize an object on every request. It's also not acceptable to cache the headers: https://frameworkbenchmarks.readthedocs.io/en/latest/Project-Information/Framework-Tests/#test-types

Reply:
I think  can use this approach and see that other platform languages are also using this approach 
 https://github.com/TechEmpower/FrameworkBenchmarks/blob/master/frameworks/Java/netty/src/main/java/hello/HelloServerHandler.java
i removed this test.